libs/common/src/sql-db-connection/migrations/1707306956373-AddUtilisationReportTables.ts

```diff
@@ -0,0 +1,95 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddUtilisationReportTables1707306956373 implements MigrationInterface {
+  name = 'AddUtilisationReportTables1707306956373';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      CREATE TABLE "AzureFileInfo" (
+        "id" int NOT NULL IDENTITY(1, 1),
+        "folder" nvarchar(255) NOT NULL,
+        "filename" nvarchar(255) NOT NULL,
+        "fullPath" nvarchar(255) NOT NULL,
+        "url" nvarchar(255) NOT NULL,
+        "mimetype" nvarchar(255) NOT NULL,
+        "utilisationReportId" int NOT NULL,
+        CONSTRAINT "PK_ea6ad35b38f9b9e248b432d1a1e" PRIMARY KEY ("id")
+      )
+    `);
+
+    await queryRunner.query(`
```

Comment:
I should remember this - is this just typeorm creating an index by default because you have a foreign key here? Semi surprised it's added 'where utilisationReportId IS NOT NULL' considering it's a non-nullable foreign key